pages/pools/pools.tsx

```diff
@@ -68,46 +69,31 @@ const Pools: NextPage = () => {
             </Link>
           </div>
         </div>
-        <table className={style.tableData}>
-          <thead className={style.tableHead}>
-            <tr>
-              <th className={style.tableHeading}>Asset</th>
-              <th className={style.tableHeading}>Total Supply</th>
-              <th className={style.tableHeading}>Supply APY</th>
-              <th className={style.tableHeading}>Total Borrowed</th>
-              <th className={style.tableHeading}>
-                <div className={style.tableHeadDiv}>
-                  Borrow APY, variable <HiOutlineInformationCircle className={style.infoSvg} />
-                </div>
-              </th>
-              <th className={style.tableHeading}>
-                <div className={style.tableHeadDiv}>
-                  Borrow APY, stable <HiOutlineInformationCircle className={style.infoSvg} />
-                </div>
-              </th>
-              <th className={style.tableHeading}></th>
-            </tr>
-          </thead>
-          <tbody>
-            {poolData && poolData.length > 0
-              ? poolData.map((pool, index) => {
-                  return (
-                    <tr className={style.tableBodyRow} key={index}>
-                      <td className={style.tableBodyDefinition}>{pool.asset}</td>
-                      <td className={style.tableBodyDefinition}>{pool.totalSupply}M</td>
-                      <td className={style.tableBodyDefinition}>{pool.supplyApy}%</td>
-                      <td className={style.tableBodyDefinition}>{pool.totalBorrowed}M</td>
-                      <td className={style.tableBodyDefinition}>{pool.borrowApyVariable}%</td>
-                      <td className={style.tableBodyDefinition}>{pool.borrowApyStable}%</td>
-                      <td className={style.tableBodyDefinition}>
-                        <button className={style.detailsBtn}>Details</button>
-                      </td>
```

Comment:
I meant to make a component for the elements inside the `return` function, something like this
```typescript
return (
    <PoolTableRow ...  />
)
```

Reply:
Done please check this